examples/src/pages/Simple.tsx

```diff
@@ -18,7 +18,7 @@ CameraControls.install({ THREE });
 
 export function Simple() {
   const canvas = useRef<HTMLCanvasElement>(null);
-  const [isLoading, setIsLoading] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
I would really prefer if it wasn't like this. Maybe add a description with a todo as to why it is done this way, and with a task to create a different trigger.